src/client/opamAdminCommand.ml

```diff
@@ -830,6 +830,100 @@ let check_command cli =
   Term.(const cmd $ global_options cli $ ignore_test_arg $ print_short_arg
         $ installability_arg $ cycles_arg $ obsolete_arg)
 
+module Compare_version_operator = struct
+  type t =
+    [ `Lt
+    | `Let
+    | `Eq
+    | `Gt
+    | `Get
+    ]
+
+  let all = [ `Lt ; `Let ; `Eq ; `Gt ; `Get ]
+
+  let eval : t -> int -> int -> bool = function
+    | `Lt -> ( < )
+    | `Let -> ( <= )
+    | `Eq -> ( = )
+    | `Gt -> ( > )
+    | `Get -> ( >= )
+
+  let assert_ t result = eval t result 0
```

Comment:
actually this section is already defined in `OpamFormula.check_relop`

Reply:
Great, thanks! I updated the code to use the relops instead of redefining them.